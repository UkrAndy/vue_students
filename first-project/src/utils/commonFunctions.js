export function sum(a, b) {
  return a + b
}

export function mult(a, b) {
  return a * b
}

export function MyFunc(x) {
  return Math.floor(10 * Math.sin(x) + 711 * Math.cos(x))
}
