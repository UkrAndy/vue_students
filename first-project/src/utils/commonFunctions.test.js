import { MyFunc } from './commonFunctions'

it('Test Mu func', () => {
  expect(MyFunc(5)).toMatchSnapshot()
})
