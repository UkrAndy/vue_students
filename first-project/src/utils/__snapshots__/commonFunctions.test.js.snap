// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Mu func 1`] = `192`;
